src/js/view/view-model.ts

```diff
@@ -63,31 +72,34 @@ class PlayerViewModel extends SimpleModel {
         }
     }
 
-    get(attr) {
+    get(attr: keyof PlayerModelAttributes | keyof MediaModelAttributes): any {
         const mediaModel = this._mediaModel;
         if (mediaModel && attr in mediaModel.attributes) {
-            return mediaModel.get(attr);
+            return mediaModel.get(attr as keyof MediaModelAttributes);
```

Comment:
would love a better way of handling this

Reply:
This is great. What do you not like?

Reply:
my understanding was this broke the type safety by manually implying to the compiler which of the union types is present, figured there might be a way to dynamically assign but i agree it's not terrible